backend/services/tool/tool_service.py

```
@@ -0,0 +1,380 @@
+            user_id: ID of the user
+            
+        Returns:
+            List of tool summaries with file_path populated
+        """
+        # Use the new discovery method
+        tool_files = self.discover_tools(repo_name, user_id)
+        summaries = []
+        
+        # Get repo base path to construct file:// URIs
+        repo_base_path = self._get_repo_base_path(user_id)
+        repo_path = repo_base_path / repo_name
```

Ruff (0.14.1)

Local variable `repo_path` is assigned to but never used

Remove assignment to unused variable `repo_path`

(F841)